### PR TITLE
fix : improve project structure

### LINKED ISSUE #18: Restructure files
- Move `CFBoard` to `lib`
- Move badly-structured files to `drafts`

### PR DESCRIPTION
**Pull request by Apllify**
closes #18 
 - Moves every single unusable file into the draft folder.
 - Generalizes the naming convention to every single file in src (outside of draft files).
 - Moves the CFBoard file into lib.

### CODE REVIEW

File: draft/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This folder will contain every single file that isn't natively integrated with the rest of the project.
```
Comment (plumsirawit) [inline draft/README.md:1]: Should we add an `s` to `draft`?

**Review by plumsirawit — CHANGES_REQUESTED** [review]
Let's use snake_case.

File: draft/README.md (hunk 1 above)
Comment (Apllify) [reply draft/README.md:1]: fixed the snake case issue

**Review by plumsirawit — APPROVED** [review]
All good.

**Pull request merged**